### DISCUSSION TITLE
Feature Request: Allow Creation of a Status-like field

### DISCUSSION THREAD

**Discussion by AMythicDev** [anyproto/contributors#59] [Anytype apps]
Anytype has a pre-biilt relation called **Status** which describes the current state of a **Task** like "To Do", "Doing" and "Done". Each task can have only one of the values in the Status field. I would like to create a relation in a similar fashion inside Anytype but it seems that it's not possible. There are some workarounds to this which I can see right now:-

**Method 1**:-
However, you can create a field which relates to a object and then manually create all the possible objects to get a similar effect but since object-related fields can have multiple objects, you do not get the facility where the field takes exactly one value. Some people might say that this is good enough but believe me when you have a ton of records it gets tedious to remove the current value and then re-add the new value.

**Method 2**:-
Similarly tags can be used to get the same effect (and with the same problems as above) one benefit you get is that you don't have to create the objects manually and they can be added on the go. But another problem arises with tags is if you want to remove/rename a value of the field, you would have to remove/rename every single instance of that tag.

If anyone wants to know why I am asking for this? I am creating a weekly planner with each task assigned to exactly one day of the week. For now, I use the first method.

So I would definitely like to see a way to create a Status-like relation in my tasks which will prove to be extremely useful in situations like this.

Thanks

**Comment by fuksman** [comment]
Hi! There is a relation type called "Status".

<img width="485" alt="image" src="https://github.com/anyproto/contributors/assets/2526238/6332726d-e4b8-4d88-9d6d-41ddd408cb9f">

**Comment by AMythicDev** [comment]
Ahh! I didn't knew the Status field type works like that. I figured out the rest. Thanks man.
